snudda/neuromodulation/neuromodulation.py

```diff
@@ -0,0 +1,126 @@
+from snudda.simulate.simulate import SnuddaSimulate
+import json
+import numpy as np
+import snudda.neuromodulation.modulation as modulation
+import snudda.neuromodulation.translator as translator
+
+
+class SnuddaNeuromodulation(SnuddaSimulate):
+
+    def __init__(self,
+                 network_path=None,
+                 network_file=None,
+                 input_file=None,
+                 verbose=False,
+                 log_file=None,
+                 disable_gap_junctions=True,
+                 simulation_config=None):
+
+        self.verbose = verbose
+        self.neuromodulation = dict()
+
+        super(SnuddaNeuromodulation, self).__init__(network_path=network_path,
+                                                    network_file=network_file,
+                                                    input_file=input_file,
+                                                    verbose=False,
+                                                    log_file=log_file,
+                                                    disable_gap_junctions=disable_gap_junctions,
+                                                    simulation_config=simulation_config)
+
+    def neuron_vector(self, vector):
+
+        return self.sim.neuron.h.Vector(vector)
+
+    def apply_neuromodulation(self, neuromodulation_file):
+
+        from pathlib import Path
+
+        define_neuro_modulation = json.load(open(Path(neuromodulation_file), 'r'))
```

Comment:
Är det bättre att använda 

with open(neuromodulation_file, "r") as f:
  define_neuro_modulation = json.load(f)

så filen stängs automatiskt på raden efteråt?

Reply:
should close file